apps/ledger-live-mobile/scripts/download-hermes-profile.mjs

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env zx
+
+import fs from "fs";
+
+// replace the package name com.ledger.live by com.ledger.live.debug in
+// AndroidManifest.xml as `react-native profile-hermes` is a bit too dumb to
+// get the right package name derived from the appIdSuffix
+const manifestPath = "./android/app/src/main/AndroidManifest.xml";
```

Comment:
a small note here, this path is platform specific and I believe it will fail on windows (using path.resolve / join would have been nice)